cpp/src/parquet/properties.h

```diff
@@ -851,6 +852,12 @@ class PARQUET_EXPORT ArrowReaderProperties {
     return coerce_int96_timestamp_unit_;
   }
 
+  void set_use_large_binary_variants(bool use_large_binary_variants) {
```

Comment:
There should be docstrings here.

Also, it's a pity we can't easily configure this per-column, as this is going to be wasteful for files with many columns...

Reply:
Hm.. that sounds bad, is there a way I can make it per column?

Reply:
Not easily I'm afraid. While `WriterProperties` has support for per-column settings, for some reason the same pattern wasn't used in other properties class...

We can omit it for now, but it's quite sub-optimal.